schema/GenomeAnnotationEnum.mcf

```diff
@@ -997,3 +997,162 @@ name: "Estonian Genome Project"
+name: "GenomeAssemblyTypeLinkedPseudohaplotypeAssemblies"
+typeOf: dcs:"GenomeAssemblyTypeEnum"
+descriptionUrl: "https://www.ncbi.nlm.nih.gov/assembly/model/#asmb_def"
+description: "A genome assembly from a diploid in which many of the haplotypic sequences have been resolved, phased and the two haplotypes have been separated. The current state of the technology is that most assemblers produce blocks that are phased, separated by blocks where the haplotype cannot be distinguished. The typical result is that the "Principal Pseudohaplotype" assembly is a mosaic of haplotype blocks linked by unresolved segments and the "Alternate Pseudohaplotype" assembly is the other haplotype wherever the haplotypes can be distinguished. A pair of pseudohaplotype assemblies derived from the same diploid individual can be linked with a cross-reference."
+
+Node: dcid:GenomeAssemblyReleaseTypeEnum
+name: "GenomeAssemblyReleaseTypeEnum"
+typeOf: schema:Class
+subClassOf: schema:Enumeration
+description: "Defines the update type of this genome assembly version release:"
+
+Node: dcid:GenomeAssemblyReleaseTypeMajor
+name: "GenomeAssemblyReleaseTypeMajor"
+typeOf: dcs:"GenomeAssemblyReleaseTypeEnum"
+descriptionUrl: "https://www.ncbi.nlm.nih.gov/assembly/help/"
+description: "Changes from the previous assembly version result in a significant change to the coordinate system. The first version of an assembly is always a major release. Most subsequent genome assembly updates are also major releases. [Field is not shown for major releases since this is the default type.]"
+
+Node: dcid:GenomeAssemblyReleaseTypeMinor
+name: "GenomeAssemblyReleaseTypeMinor"
+typeOf: dcs:"GenomeAssemblyReleaseTypeEnum"
+descriptionUrl: "https://www.ncbi.nlm.nih.gov/assembly/help/"
+description: "Changes from the previous assembly version are limited to the following changes, none of which result in a significant change to the coordinate system of the primary assembly-unit: adding, removing or changing a non-nuclear assembly-unit; dropping unplaced or unlocalized scaffolds; adding up to 50 unplaced or unlocalized scaffolds which are shorter than the current scaffold-N50 value; or replacing a component with a gap of the same length."
+
+Node: dcid:GenomeAssemblyReleaseTypePatch
+name: "GenomeAssemblyReleaseTypePatch"
+typeOf: dcs:"GenomeAssemblyReleaseTypeEnum"
+descriptionUrl: "https://www.ncbi.nlm.nih.gov/assembly/help/"
+description: "The only change from the previous assembly version is the addition or modification of a patch assembly-unit (relevant for assemblies maintained by the Genome Reference Consortium )"
+
+Node: dcid:GenomeAssemblyLevelEnum
+name: "GenomeAssemblyLevelEnum"
+typeOf: schema:Class
+subClassOf: schema:Enumeration
+description: "The highest level of assembly for any object in the assembly."
+descriptionUrl: "https://www.ncbi.nlm.nih.gov/assembly/help/#level"
+
+Node: dcid:GenomeAssemblyLevelComplete_genome
+name: "GenomeAssemblyReleaseTypeComplete_genome"
```

Comment:
camel case?

Reply:
Converted all enumeration dcids and names in t